content/outcomes/application-observability/tsm-slo-based-measurement-and-monitoring-apps.md

```diff
@@ -0,0 +1,85 @@
+An SLO consists of one or multiple service level indicators (SLIs). SLOs defined using a combination of SLIs allow teams to describe service or application health in a more precise and relevant way.  
+
+SLIs capture important low-level performance characteristics for a particular service. SLI metrics are frequently captured and aggregated over a short time window, such as every 15 seconds. Examples of SLIs would be:
+  *	99 percent of successful requests respond with latencies faster than 350 ms (99th percentile latency < 350 ms)
+  * The service responds with error codes for fewer than 0.1 percent of requests (error rate < 0.1%)
+
+![Tanzu Service Mesh Service SLO view](/images/outcomes/app-observability/tsm-service-slo-view.png)
+
+
+## Use Cases
+
+Tanzu Service Mesh provides an interface where you can configure SLO targets and select the SLIs that determine whether the targets are met. Metric levels are displayed in real-time graphs in the Tanzu Service Mesh Console user interface.  
+
+1. **Apply SLO to Services** - You can group services with common characteristics or that are part of one or more related applications in a namespace to form a service group in Tanzu Service Mesh. You can configure and set service groups to automatically track against the same SLO definition. [more](https://docs.vmware.com/en/VMware-Tanzu-Service-Mesh/services/slos-with-tsm/GUID-A13A6444-D48B-40D1-BF52-6151A44D6CEA.html)
+
+1. **View SLO and SLI Violations** - After you define SLO configurations, you can view and track violations of SLIs and SLOs in real time in the Tanzu Service Mesh Console user interface. [more](https://docs.vmware.com/en/VMware-Tanzu-Service-Mesh/services/slos-with-tsm/GUID-F114FF5C-E632-4711-9D5F-B25FD980C5EF.html)
+
+1. **Track “Error Budgets”** - Based on SLOs, teams can track “error budgets,” that is, a length of time over which performance can be outside the desired SLIs without violating the SLO. [more](https://docs.vmware.com/en/VMware-Tanzu-Service-Mesh/services/slos-with-tsm/GUID-9C1D70D7-1C0C-4F72-A39C-42EDCDBB5D9E.html)
+
+1. **Track Autoscaling Impact on Services** - Developers and operators can use the Service Level Objective (SLO) graph to see how Tanzu Service Mesh Service Autoscaler impacts the stated microservice objectives. [more](https://docs.vmware.com/en/VMware-Tanzu-Service-Mesh/services/service-autoscaling-with-tsm-user-guide/GUID-E5684E2C-8D32-4422-BA3B-822606D01E6C.html)
+
+
+## Best Practices
+
+1. **Set High-Level Objectives for Production Environments** – A common measure of the health of a service are latency percentile values. Of the three latency percentiles available to use with Tanzu Service Mesh SLOs, the p99 latency is the most appropriate service level indicator (SLI) for production environments because this metric most fully covers the experience of the users of a service. [more](https://docs.vmware.com/en/VMware-Tanzu-Service-Mesh/services/slos-with-tsm/GUID-2651936B-E8C7-41C3-AE76-5163611970C3.html)
```

Comment:
I might say 99th percentile (p99) the first time, so that readers who aren't familiar with the p99 notation know what you're talking about.

Reply:
fixed